src/Resources/translations/messages.pl.yml

```diff
@@ -1,20 +1,17 @@
 bitbag:
     ui:
-        shipping_gateways: Bramy dostawców
-        create_shipping_gateway: Stwórz bramę dostawcy
-        manage_shipping_gateways: Zarządzaj bramami dostawców
         choose_shipping_method: Wybierz metodę wysyłki
+        create_shipping_gateway: Stwórz bramę dostawcy
+        download_shipping_label: Pobierz etykietę
         export_all_new_shipments: Eksportuj wszystkie nowe przesyłki
-        no_shipping_label: Brak etykiety
         export_shipments: Eksportuj przesyłki
-        download_shipping_label: Pobierz etykietę
-        shipping_exports: Eksportuj przesyłki do zewnętrznego API
+        exported: Eksportowano
+        exported_at: Eksportowano
```

Comment:
Eksportowano -> Wyeksportowano? Because "czas dokonany" (not sure how to describe in EN)

Reply:
Fixed